.github/workflows/publish.yml

```diff
@@ -17,4 +17,24 @@ jobs:
           DOCKER_USER: ${{secrets.DOCKER_USER}}
           DOCKER_PASSWORD: ${{secrets.DOCKER_PASSWORD}}
       - name : push to Docker Hub
-        run: make build-image image-push
\ No newline at end of file
+        run: make build-image image-push
+  push-artifact-hub:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check the repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: azure/setup-helm@v3
+        with:
+          version: v3.10.1
```

Comment:
You will change the version of helm (like the version of k8s) trough time ?

Reply:
Yes, but the required version will not change unless we need a new option

Reply:
Ok